dist/main.js

```diff
@@ -99,7 +99,7 @@ class BotFrameworkInstrumentation {
             body: {
                 "documents": [
                     {
-                        "language": "en",
+                        "language": session.preferredLocale().substring(0, 2) || "en",
```

Comment:
@iMicknl - thanks for the update, just please make sure there are no nulls/undefined with:
session; session.preferredLocale()
(Here and below)